Sources/Playbook/Components/Label Pill/PBLabelPill.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  Playbook Swift Design System
+//
+//  Copyright © 2024 Power Home Remodeling Group
+//  This software is distributed under the ISC License
+//
+//  PBLabelPill.swift
+//
+
+import SwiftUI
+
+public struct PBLabelPill: View {
+  let variant: PBPill.Variant
+  let labelValue: String?
+  let pillValue: String?
+  let labelFontSize: PBFont
+  let labelColor: Color
+  let offset: Double?
+  let horizontalPadding: CGFloat?
+  public init(
+    variant: PBPill.Variant = .neutral,
+    labelValue: String? = "Service Needed",
+    pillValue: String? = "76",
+    labelFontSize: PBFont = .caption,
```

Comment:
I think the font is always caption. This prop is never being called. Guess we can remove that.

Reply:
Ok, I was thinking we could keep this so users have the choice?